src/components/text/Text.tsx

```diff
@@ -18,12 +18,19 @@ type OtherText = (typeof textOthers)[number];
 
 export const textOptions = [
   {
-    label: 'Page Header (3XL) · 34px (2.125rem)',
+    label: 'Page Header (3XL) · 30px (1.875rem)',
```

Comment:
I think this should be 32px according to the design

Reply:
I just copied text straight from the design 😆 
<img width="518" alt="image" src="https://github.com/Zonos/amino/assets/17950626/4a4f3304-c917-472c-be7d-816e308942b8">

Reply:
https://www.figma.com/design/WnKnmG7L3Q74hqPsw4rbEE/Amino-2.0?node-id=2100-71774&m=dev

Reply:
interesting 
<img width="936" alt="image" src="https://github.com/Zonos/amino/assets/50718218/54c33bc1-cf38-4e25-b839-81c8b98d9798">

Reply:
Interesting 🤔 That's weird 😆